meta-mender-update-modules/conf/layer.conf

```diff
@@ -11,4 +11,4 @@ BBFILE_COLLECTIONS += "mender-update-modules"
 BBFILE_PATTERN_mender-update-modules = "^${LAYERDIR}/"
 BBFILE_PRIORITY_mender-update-modules = "15"
 
-LAYERSERIES_COMPAT_mender-update-modules = "kirkstone"
\ No newline at end of file
+LAYERSERIES_COMPAT_mender-update-modules = "mickledore"
```

Comment:
Why all three branches on the others, and not here?

Reply:
That was a borderline decision, because the new Application Update Module requires `mickledore`. I'm fine with either for now, but as I expect the branch to move forwards over the next months and converge to the next LTS release, it is semi-temporary anyways. If you think marking it for all three, and accepting the UM to break for non-`mickledore` builds I can do that @kacf

Reply:
No, that's a good call, I was just curious!